Control/public/Model.js

```diff
@@ -297,14 +297,25 @@ export default class Model extends Observable {
   }
 
   /**
-   * After pages load check if the user enabled notifications
+   * Request notification permission
    */
-  checkBrowserNotificationPermissions() {
+  requestBrowserNotificationPermissions() {
```

Comment:
I would recommend you make use of the `check` method below to see if notifications are granted or not as you already have it.